mpp/include/mpp_io_misc.inc

```diff
@@ -177,10 +177,8 @@
       if( pe.EQ.mpp_root_pe() )then
           iunit = stdlog()  ! PGI compiler does not like stdlog() doing I/O within write call
           write( iunit,'(/a)' )'MPP_IO module '//trim(version)
-#ifdef use_netCDF
           text = NF_INQ_LIBVERS()
           write( iunit,'(/a)' )'Using netCDF library version '//trim(text)
-#endif
       endif
 
 #ifdef CRAYPVP
```

Comment:
IIRC, this requires the use of the FFIO system and I don't think that is supported anymore.

Reply:
I still see the `assign` command and library call in the Cray PE (Fortran Reference Manual)[https://pubs.cray.com/content/S-3901/9.1/cray-fortran-reference-manual/enhanced-io-using-the-assign-environment].  However, I was unable to test if it works due to CCE fortran not linking, and the latest CCE version on Gaea does not have the Fortran compiler.  This is something we should look into.  I'll create a new issue.

Reply:
Issue #248 created.